test/object-store/sync/flx_sync.cpp

```diff
@@ -3299,6 +3300,79 @@ TEST_CASE("flx: bootstrap changesets are applied continuously", "[sync][flx][app
     CHECK(user_commit_version == bootstrap_version + 1);
 }
 
+TEST_CASE("flx: async open + register subscription callack while bootstrapping", "[sync][flx][async-open]") {
+    std::mutex mutex_task_completed;
+    std::condition_variable cv;
+    std::atomic<bool> async_open_invoked = false;
+    std::atomic<bool> subscription_invoked = false;
+    std::atomic<bool> async_open_completed = false;
+    FLXSyncTestHarness harness("flx_bootstrap_batching", {g_large_array_schema, {"queryable_int_field"}});
```

Comment:
You should just use `FLXSyncTestHarness harness("flx_bootstrap_batching");` here, no need for the large schema.